calculator.js

```diff
@@ -4,6 +4,19 @@ let lastButtonClicked = null;
 
 let button = document.getElementById("result");
 
+let numBtnElem = document.querySelectorAll("[data-number]")
+
+numBtnElem.forEach(function(button) {
+    button.addEventListener('click', function() {
+        (lastButtonClicked === "Operator") result.textContent = "";
```

Comment:
Can you explain what this line doing?

Reply:
Firstly 
there is if  in the beginning of it it I do not know how it fly
this line is to clear the result div after clicking on an opertaor button 
thanks

Reply:
should I do add data attribute to the operator buttons .... etc (loop) ?

Reply:
Yes, you can refactor the code of the + - / and * buttons following the same approach.